pkg/filterengine/filters/namespace_checker.go

```diff
@@ -56,7 +56,8 @@ func (f NamespaceChecker) Run(object interface{}, event *events.Event) {
 	}
 	if botkubeConfig != nil {
 		for _, resource := range botkubeConfig.Resources {
-			if resource.Name == strings.ToLower(event.Kind) {
+			resourceName := strings.Split(strings.ToLower(event.Title), " ")[0]
+			if resource.Name == resourceName {
```

Comment:
@hellozee we have a helper function implemented which returns GVR from given Kind https://github.com/infracloudio/botkube/blob/develop/pkg/utils/utils.go#L258. We can use that method to build "group/version/resource" or "version/resource" (if the group is nil) check

Reply:
This is not working for delete events since the resource type return by that method is `/`. Here is how I am trying to do it.
```go
var eventObj *coreV1.Event
err := utils.TransformIntoTypedObject(object.(*unstructured.Unstructured), &eventObj)
if err != nil {
    log.Errorf("Unable to transform object type: %v, into type: %v", reflect.TypeOf(object), reflect.TypeOf(eventObj))
    return
}
tempResourceName, _ := utils.GetResourceFromKind(eventObj.InvolvedObject.GroupVersionKind())
resourceName := utils.GVRToString(tempResourceName)
```

Reply:
@hellozee let's do this
1. Add a new field `Resource` in the Event struct
2. Set the Resource value while creating New Event
3. Refer event.Resource while building Title as well as in filters
Sounds good?

Reply:
Cool, will do that instead, 🐱

Reply:
@hellozee will you be able to implement the suggested changes?

Reply:
Ahh sorry, completely forgot about it, will do by this weekend, 😸

Reply:
Hey @hellozee, please let me know if you are working on this.

Reply:
whoops! expect a PR by evening for sure, 🙃